x/nft/spec/01_concepts.md

```diff
@@ -0,0 +1,13 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Class
+
+`x/nft` module defines a struct `Class` to describe the common characteristics of a class of nft, under this class, you can create a variety of nft, which is equivalent to an erc721 contract for Ethereum. The design is defined in the [ADR 043](../../../docs/architecture/adr-043-nft-module.md).
+
+## NFT
+
+The full name of NFT is Non-Fungible Tokens. Because of the irreplaceable nature of NFT, it means that it can be used to represent unique things. The nft implemented by this module is fully compatible with Ethereum erc721.
```

Comment:
```suggestion
The full name of NFT is Non-Fungible Tokens. Because of the irreplaceable nature of NFT, it means that it can be used to represent unique things. The nft implemented by this module is fully compatible with Ethereum ERC721 standard.
```